practice-app/website/api/event.py

```diff
@@ -56,17 +60,16 @@ def event():
                 return "Try Later", 403
```

Comment:
I think 403 is a wrong response code because it means "forbidden". if there is an error during db operations 503 is more appropriate

Reply:
Agreed related discussion [here](https://github.com/bounswe/2021SpringGroup6/issues/90)

Reply:
This issue does not exist after merging the master into this branch, and solved after resolving the conflicts as well.